.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger Buildkite Pipeline
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Trigger a Buildkite Build using GHA arguments'
+        uses: "buildkite/trigger-pipeline-action@feature/support-action-inputs"
```

Comment:
I think this won't be able to get merged right? Because the branch will be deleted after this PR is merged

I'm wondering if bats is still the best way to test this plugin. I'm not sure how complex that is but I think it might be "safer" so we don't have to rely on this GH action requiring a specific buildkite pipeline and token tied to a user. 
And we can always do an integration test which it looks like what youve done with https://github.com/lizrabuya/kite-flying-tutorial/blob/main/.github/workflows/fly-delta.yml already 👍

Reply:
@jradtilbrook , yeah that too is another problem. This workflow is ideally invoked for development as mention in the readme using the `act ` app. I will raise another PR to remove this workflow then.